ibm_watson/assistant_v2.py

```diff
@@ -1215,17 +1363,19 @@ class LocaleEnum(Enum):
 
 class MessageContextSkill():
     """
-    Contains information specific to a particular skill used by the Assistant.
+    Contains information specific to a particular skill used by the Assistant. The
+    property name must be the same as the name of the skill (for example, `main skill`).
 
     :attr dict user_defined: (optional) Arbitrary variables that can be read and
           written by a particular skill.
-    :attr dict system: (optional) System context data used by the skill.
+    :attr MessageContextSkillSystem system: (optional) System context data used by
+          the skill.
     """
 
     def __init__(self,
                  *,
                  user_defined: dict = None,
-                 system: dict = None) -> None:
+                 system: 'MessageContextSkillSystem' = None) -> None:
```

Comment:
Was this the change we were avoiding?

Reply:
ah I thought I removed it everywhere, good catch.